src/Widgets/Common/ImageCarouselWidget.php

```diff
@@ -42,6 +42,11 @@ public function getSettings()
         $settings->createCustomClass();
         $settings->createAppearance();
 
+        $settings->createCheckbox('fullWidth')
+            ->withDefaultValue(true)
```

Comment:
Ich denke wir sollten die Einstellung per default auf false setzen, da sonst bestehende Kunden beim Inhalte neu generieren plötzlich einen anderen Content haben.

```suggestion
            ->withDefaultValue(false)
```